src/main/java/org/apache/maven/report/projectinfo/dependencies/renderer/DependenciesRenderer.java

```diff
@@ -591,16 +595,75 @@ private void renderSectionDependencyFileDetails() {
                         fileLength = "-";
                     }
 
-                    tableRow(hasSealed, new String[] {
-                        name,
-                        fileLength,
-                        String.valueOf(jarDetails.getNumEntries()),
-                        String.valueOf(jarDetails.getNumClasses()),
-                        String.valueOf(jarDetails.getNumPackages()),
-                        jdkRevisionCellValue,
-                        debugInformationCellValue,
-                        sealedCellValue
-                    });
+                    if (jarDetails.isMultiRelease()) {
+                        String htmlBullet = "&#160;&#160;&#160;&#x2022; ";
+                        String rootTag = htmlBullet + getI18nString("file.details.multirelease.root");
+                        String versionedTag = htmlBullet + getI18nString("file.details.multirelease.versioned");
+
+                        // general jar information row
+                        tableRow(hasSealed, new String[] {
+                            name,
+                            fileLength,
+                            String.valueOf(jarDetails.getNumEntries()),
+                            "",
+                            "",
+                            "",
+                            "",
+                            sealedCellValue
+                        });
+
+                        JarVersionedRuntimes versionedRuntimes = jarDetails.getVersionedRuntimes();
+                        Collection<JarVersionedRuntime> versionedRuntimeList =
+                                versionedRuntimes.getVersionedRuntimeMap().values();
+
+                        // workaround to count the number of root content entries
```

Comment:
Please add the ticket ref for MSHARED here.

Reply:
ok